drivers/counter/timer_dtmr_cmsdk_apb.c

```diff
@@ -128,21 +176,28 @@ static const struct counter_driver_api timer_dtmr_cmsdk_apb_api = {
 	.stop = timer_dtmr_cmsdk_apb_stop,
 	.read = timer_dtmr_cmsdk_apb_read,
 	.set_alarm = timer_dtmr_cmsdk_apb_set_alarm,
+	.disable_alarm = timer_dtmr_cmsdk_apb_disable_alarm,
+	.set_wrap = timer_dtmr_cmsdk_apb_set_wrap_alarm,
 	.get_pending_int = timer_dtmr_cmsdk_apb_get_pending_int,
+	.get_wrap = timer_dtmr_cmsdk_apb_get_wrap,
 };
 
 static void timer_dtmr_cmsdk_apb_isr(void *arg)
 {
 	struct device *dev = (struct device *)arg;
 	const struct timer_dtmr_cmsdk_apb_cfg * const cfg =
 						dev->config->config_info;
+	struct timer_dtmr_cmsdk_apb_dev_data *data = dev->driver_data;
 
 	cfg->dtimer->timer1intclr = DUALTIMER_INTCLR;
-	if (user_cb) {
+	if (data->user_cb) {
 		/* user_data paramenter is not used by this driver */
```

Comment:
seems like missing implementation. It's software feature so should be easily supported.